config/locales/translations/zendesk_apps_support.yml

```diff
@@ -119,6 +119,16 @@ parts:
       key: "txt.apps.admin.error.app_build.invalid_requirements_types.other"
       title: "App builder job: requirements file contains invalid types error"
       value: "requirements.json contains invalid types: %{invalid_types}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.unsupported_mime_type_detected.one"
+      title: "App builder job: directory contains unsupported mime type"
```

Comment:
Can you add following to the title please?

> MIME is an abbreviation for Multipurpose Internet Mail Extensions. See https://en.wikipedia.org/wiki/MIME

Reply:
Sure can. Thank you, Hiro.